apis/v1alpha1/types.go

```diff
@@ -76,15 +76,18 @@ func (s *AuthorizationPolicySpec) GetFailurePolicy() admissionregistrationv1.Fai
 
 // Authorization defines an authorization policy rule
 type Authorization struct {
-	// Expression represents the expression which will be evaluated by CEL.
+	// Match represents the match condition which will be evaluated by CEL. Must evaluate to bool.
+	// +optional
+	Match string `json:"match"`
```

Comment:
It should have `omitempty` (wondering if it makes sense to use a pointer)

Reply:
don't see any advantage as an empty string should work well too.